### PR TITLE
Allows guns and other security stuff to be suit slotted on the ablative trench coat

### PR DESCRIPTION
**Pull request by Bm0n**
## What Does This PR Do
Lets you put guns and batons in the suit slot of the ablative trench coat

The allowed items list is copypasted from the regular armour vest

## Why It's Good For The Game
Basically, I forgot to add the allowed items list for the ablative trench coat when porting it over from TG. Oops!

## Testing

Tested locally

## Changelog
:cl: Bmon
fix: You can now put weapons and other security items in the suit slot of the ablative trench coat
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -513,6 +513,7 @@
 	body_parts_covered = UPPER_TORSO | LOWER_TORSO | LEGS | FEET | ARMS | HANDS
 	cold_protection = UPPER_TORSO | LOWER_TORSO | LEGS | FEET | ARMS | HANDS
 	heat_protection = UPPER_TORSO | LOWER_TORSO | LEGS | FEET | ARMS | HANDS
+	allowed = list(/obj/item/gun/energy,/obj/item/reagent_containers/spray/pepper,/obj/item/gun/projectile,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/restraints/handcuffs,/obj/item/flashlight/seclite,/obj/item/melee/classic_baton/telescopic,/obj/item/kitchen/knife/combat)
```
Comment (lewcc) [inline code/modules/clothing/suits/miscellaneous.dm:516]:
```suggestion
	allowed = list(/obj/item/gun/energy, /obj/item/reagent_containers/spray/pepper, /obj/item/gun/projectile, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/flashlight/seclite, /obj/item/melee/classic_baton/telescopic, /obj/item/kitchen/knife/combat)
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -513,6 +513,7 @@
 	body_parts_covered = UPPER_TORSO | LOWER_TORSO | LEGS | FEET | ARMS | HANDS
 	cold_protection = UPPER_TORSO | LOWER_TORSO | LEGS | FEET | ARMS | HANDS
 	heat_protection = UPPER_TORSO | LOWER_TORSO | LEGS | FEET | ARMS | HANDS
+	allowed = list(/obj/item/gun/energy, /obj/item/reagent_containers/spray/pepper, /obj/item/gun/projectile,/obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/flashlight/seclite, /obj/item/melee/classic_baton/telescopic, /obj/item/kitchen/knife/combat)
```
Comment (Sirryan2002) [inline code/modules/clothing/suits/miscellaneous.dm:516]: does this mean shotguns can be put on the coat?

Comment (Bm0n) [reply code/modules/clothing/suits/miscellaneous.dm]: Should now, yeah.

**Review by Sirryan2002 — APPROVED** [review]
code is fine

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**